### ISSUE TITLE
Character's inventory not available

### ISSUE THREAD

**Issue by Slickwilli21** [dasilva333/TowerGhostForDestiny#493]
The vault has its inventory but each character does not have anything show up in their inventory. If you move something from the vault to that character it then shows up in the character inventory. Any ideas? I have cleared the cache and uninstalled/reinstalled.

Maybe something to do with the latest bungie update?

**Comment by z929669** [comment]
I am also having issues with certain characters and gear types. For example, my Titan and Hunter have light=0 and no gear or inventory, whilst my Warlock does. The android version displays correctly as does the Bungie app, so maybe the FF extension API is not functioning correctly?

Also, refresh does not work for item slots having no inventory.

**Comment by slick9901** [comment]
I am also having the same issues as of last night.  At some point I tried to transfer my raid gear loadout to another character and the transfer seemed to freeze in the middle of the transfer.  Since then I have been unable to see anything on each of my individual characters.  All that shows up is whatever is in the vault.  I have cleared cache, uninstalled/re-installed, refreshed, cleared filters and still the same result.  I am unsure whether it has to do with the transfer hanging in the middle or not.  Just something I noticed.  The result is now the same on both my android tablet and on my BlackBerry.  On the firefox app however everything is showing correctly.  Hope this helps.
